environments/echis/public.yml

```diff
@@ -81,3 +81,7 @@ localsettings:
   ENABLE_PRELOGIN_SITE: False
   ENTERPRISE_MODE: True
   RESTRICT_DOMAIN_CREATION: True
+  STATIC_TOGGLE_STATES:
+    mobile_ucr_linked_domain:
+      always_enabled:
+        - 'fmoh-echis-staging'
```

Comment:
Hey Ethan, just FYI, we are moving toggle state declaration from HQ code to env config. This is the only eCHIS toggle I found, I think this domain exists only on Prod but not eCHIS env, but if it is, then they need to do an update-config with this change

@esoergel